group_vars/all/vars.yml

```diff
@@ -1,4 +1,12 @@
 ---
+
+# GitHub settings
+github_token: '{{ lookup("env", "ANSIBLE_GITHUB_TOKEN") | default("") }}'
+api_base: https://api.github.com
+deployments_endpoint: '{{api_base}}/repos/{{ repo }}/deployments'
+deploy_environment: '{{ staging|default(None) or qa|default(None) or production }}'
+deploy_description: 'Deploying {{ gitref }} to {{ deploy_environment }}'
```

Comment:
Would it be hard to also include hostname here?

Reply:
As nearly as I can tell, those don't actually get used for the Slack notification. But I can include it anyway, since it would be a record for us that we could query if we needed to do so.